adoc/Private-Registry-Installation.adoc

```diff
@@ -600,6 +600,10 @@ spec:
     image: registry.suse.com/harbor/harbor-redis:2.1.0
     customConfig:
       - "dir /data"
+    securityContext:
```

Comment:
As discussed, this can be fixed by updating our images to use the default UID/GID value of 1000 instead of exposing this to our users. This is valid not only for Harbor, but in general, for anyone who wants to use the SUSE Redis operator.

Reply:
We agreed to solve this by documentation for now. This whole CRD configuration will be "hidden" in future updates by bringing it under the umbrella of the Harbor helm chart, so the user will exposed to even less complexity.